plugins/auth/auth_base.js

```diff
@@ -101,20 +101,28 @@ exports.check_user = function (next, connection, credentials, method) {
         return;
     }
 
-    const passwd_ok = function (valid, message) {
+    // valid: (true|false)
+    // opts: ({ message, code }|String)
+    var passwd_ok = function (valid, opts) {
```

Comment:
Switching from `const` to `var`  is part of why your lint tests are failing. Better still is shedding the function assignment and creating a function declaration: `function passwd_ok (valid, opts) {`.